app/models/stock_level.rb

```diff
@@ -23,7 +23,7 @@ def path(params, instance = nil)
         if params[:filter] && params[:filter].key?(:skus)
```

Comment:
This entire override of the path method can now be deleted.  It was there simply to "frig" the filter into the way that we were doing it.  We shouldn't need this at all now it is done to the json api standard